### DISCUSSION TITLE
Running the notebook as a script?

### DISCUSSION THREAD

**Discussion by Linardos** [FETS-AI/Challenge#53] [Q&A] [answered]
Is there a way to run the code from the notebook (FeTS_Challenge.ipynb) as a script ? As I'm working from a server and it takes quite a bit of time to run it would be ideal to be able to have it run with a script using tmux

**Comment by sarthakpati** [comment, marked as answer]
Hi @Linardos,

Thank you for your interest in the FeTS Challenge! You can run any jupyter notebook as a script using the following mechanism:

```bash
pip3 install runipy 
runipy notebook_to_run.ipynb
```

Cheers,
Sarthak

**Reply by Linardos** [reply]
Thanks for the quick reply! Has this been tested to work with the challenge script? In my case I get an obscure error:
![image](https://user-images.githubusercontent.com/26694607/123456765-e3c06600-d5eb-11eb-980f-9da029eff9fa.png)
This does not happen when I run the same notebook normally

**Reply by sarthakpati** [reply]
Nope, I haven't tried running the challenge script this way, I just provided a generic mechanism of running a notebook as a script.

Pinging @brandon-edwards and @psfoley for clarity.

**Reply by brandon-edwards** [reply]
I have not tried this. I know VSCode also provides conversion from a notebook to a module, but a foolproof way would also be to simply copy and past the relevant cells into a .py file.

**Reply by Linardos** [reply]
my bad, it was actually a problem with the script because I had changed the used partition csv to "small_split.csv". Apparently the problem comes from there. runipy works fine.

**Reply by sarthakpati** [reply]
Glad it worked!
